lib/licenses.nix

```diff
@@ -246,6 +246,12 @@ in mkLicense lset) ({
     free = false;
   };
 
+  cc-by-nc-nd-40 = {
+    spdxId = "CC-BY-NC-ND-4.0";
+    fullName = "Creative Commons Attribution Non Commercial No Derivatives 4.0 International";
+    free = false;
+  };
```

Comment:
This got added in the meantime, so to fix eval it needs to be dropped again

Reply:
Gotcha. I couldn't make sense of the failing CI.

Rebased and dropped the commit now.